adminmaintain.html

```diff
@@ -2,7 +2,7 @@
 <html>
```

Comment:
You have done good work but it would be better if you separated your work into folders i.e `css` folder to contain your css, `html` folder to contain your html and `js` to contain your JavaScript.

As per the instructions in your challenge one, *You were advised to put all your code in the UI folder *